cms/djangoapps/contentstore/rest_api/v2/views/downstreams.py

```diff
@@ -137,6 +142,31 @@ def get(self, request: _AuthenticatedRequest, course_key_string: str):
         return Response(serializer.data)
 
 
+@view_auth_classes()
+class DownstreamContextListView(DeveloperErrorViewMixin, APIView):
+    """
+    Serves library block->downstream usage keys
+    """
+    def get(self, request: _AuthenticatedRequest, usage_key_string: str) -> Response:
+        """
+        Fetches downstream links for given publishable entity
+        """
+        try:
+            usage_key = UsageKey.from_string(usage_key_string)
+        except InvalidKeyError as exc:
+            raise ValidationError(detail=f"Malformed usage key: {usage_key_string}") from exc
+
+        downstream_usage_key_list = (
+            PublishableEntityLink
+            .get_by_upstream_usage_key(upstream_usage_key=usage_key)
+            .values_list("downstream_usage_key", flat=True)
+        )
+
+        downstream_usage_key_str_list = [str(usage_key) for usage_key in downstream_usage_key_list]
+
+        return Response(downstream_usage_key_str_list)
```

Comment:
NIT: Instead of manually doing this, we can use serializer.

Reply:
I avoided it because I was returning only a list of strings.
I updated to use a serializer here (fb6303cad7746911c423a1feea8ea0ad6e42dc63) so we can extend it further in the future if needed.